core/src/main/java/org/lionweb/lioncore/java/utils/ModelValidator.java

```diff
@@ -0,0 +1,45 @@
+package org.lionweb.lioncore.java.utils;
+
+import org.lionweb.lioncore.java.metamodel.Metamodel;
+import org.lionweb.lioncore.java.model.Model;
+
+public class ModelValidator extends Validator<Model> {
+    @Override
+    public ValidationResult validate(Model element) {
+        throw new UnsupportedOperationException();
+    }
+
+    /**
+     * Eventually the Metamodel will be also a Model so MetamodelValidator will delegate to ModelValidator.
+     * For now, we expose this method instead.
+     *
+     * Ids can only contain these symbols:
+     *
+     * - lowercase latin characters: a..z
+     * - uppercase latin characters: A..Z
+     * - arabic numerals: 0..9
+     * - underscore: _
+     * - hyphen: -
+     */
+    public static boolean isValidID(String id) {
```

Comment:
Can't this be a static method on some other class? Then the `ModelValidator` class is essentially superfluous.

Reply:
Ok, I will move it to MetamodelValidator and recreate ModelValidator later on